### PR TITLE
[RHACS] [rhacs-docs-4.4] ROX-23460: Adding 4.4.1 RN

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.4

Issue:
https://issues.redhat.com/browse/ROX-23460

Link to docs preview:
https://74570--ocpdocs-pr.netlify.app/openshift-acs/latest/release_notes/44-release-notes#resolved-in-version-441_release-notes-44

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-4.4`

### CODE REVIEW

File: release_notes/44-release-notes.adoc
Comment (ovalenti) [inline release_notes/44-release-notes.adoc:585]:
```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the Power architecture where CORE_BPF collection method is not functioning properly. From {product-title-short} 4.4.1 and later, in this particular case, Collector uses automatically the EBPF collection method when CORE_BPF is selected.

Impact: If you use OpenShift Container Platform 4.12 on the Power architecture and in offline mode, you must use a support package.
```

**Review by ovalenti — CHANGES_REQUESTED** [review]
I expanded slightly to give more context, and replaced the term `workaround` since this known issue is about the limitations of a workaround; which seemed a bit misleading.

**Comment by agantony** [comment]
> I expanded slightly to give more context, and replaced the term `workaround` since this known issue is about the limitations of a workaround; which seemed a bit misleading.

@ovalenti  I have incorporated your suggestions. If the changes look good to you, please approve the PR. Thanks!

**Review by ovalenti — APPROVED** [review]
The "known issues" section looks good to me, thanks 👍 

I read through the other parts and it seems alright, but I cannot validate the content.

File: release_notes/44-release-notes.adoc
Comment (dhaus67) [inline release_notes/44-release-notes.adoc:573]: I thought we do not document this - @dcaravel to verify whether this should be here or not.

**Review by dhaus67 — APPROVED** [review]
The rest of the release note section looks good to me; I have one comment regarding the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` - pinged @dcaravel to confirm, please do not merge without his approval.

File: release_notes/44-release-notes.adoc
Comment (dcaravel) [reply release_notes/44-release-notes.adoc:573]: That's correct, please remove the `ROX_UNQUALIFIED_SEARCH_REGISTRIES` section for now. Thanks @agantony

Comment (agantony) [reply release_notes/44-release-notes.adoc]: ack

**Review by dcaravel — APPROVED** [review]

File: release_notes/44-release-notes.adoc
Comment (kcarmichael08) [inline release_notes/44-release-notes.adoc:587]:
```suggestion
* Currently, there is an issue for OpenShift Container Platform 4.12 on the IBM Power architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected. 
```

File: release_notes/44-release-notes.adoc
Comment (kcarmichael08) [inline release_notes/44-release-notes.adoc:589]:
```suggestion
If you are using OpenShift Container Platform 4.12 on the IBM Power architecture and in offline mode, you must use a support package.
```

**Review by kcarmichael08 — COMMENTED** [review]
Couple of minor things, otherwise lgtm!

**Comment by kcarmichael08** [comment]
This is ready to merge, but I think we need to wait for Monday 4/15 until the release goes out.

**Pull request merged**
